lib/commons/dom/is-focusable.js

```diff
@@ -4,15 +4,14 @@ import AbstractVirtualNode from '../../core/base/virtual-node/abstract-virtual-n
 import { getNodeFromTree } from '../../core/utils';
 
 /**
- * Determines if an element is focusable
+ * Determines if an element is keyboard or programmatically focusable.
  * @method isFocusable
  * @memberof axe.commons.dom
  * @instance
  * @param {HTMLElement} el The HTMLElement
  * @return {Boolean} The element's focusability status
  */
-
-function isFocusable(el) {
+export default function isFocusable(el) {
```

Comment:
I wonder if maybe we should add some option here to say `isFocusable(el, { noNegativeTabindex: true })`?

This seem like sort of a common occurance.

Reply:
I agree. Though maybe the option could be called `programmatically`?

```
const focusable = isFocusable(el, { programmatically: false })
```

Reply:
https://github.com/dequelabs/axe-core/issues/3500